### PR TITLE
Stats ctr global logmsg allocated size in bytes

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: lib/stats/stats-cluster-single.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2002-2017 Balabit
```
Comment (kvch) [inline lib/stats/stats-cluster-single.c]: Please, fix the years in this header.

File: lib/stats/stats-cluster-single.h [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2002-2017 Balabit
```
Comment (kvch) [inline lib/stats/stats-cluster-single.h]: Copyright header, too.

**Comment by pzoleex** [comment]
@lbudai the test are on this branch: pzolee-stats-global-allocated-bytes

**Comment by lbudai** [comment]
@kira-syslogng test this please test branch=pzolee-stats-global-allocated-bytes;

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by lbudai** [comment]
@kira-syslogng test this please test branch=pzolee-stats-global-allocated-bytes;

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by lbudai** [comment]
@kira-syslogng test this please test branch=pzolee-stats-global-allocated-bytes;

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by pzoleex** [comment]
@kira-syslogng test this please test branch=pzolee-stats-global-allocated-bytes;

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests passed on testdb branch: pzolee-stats-global-allocated-bytes and test suite: functions

**Pull request merged**